packages/flutter/lib/src/material/snack_bar_theme.dart

```diff
@@ -5,10 +5,7 @@
 import 'dart:ui' show lerpDouble;
 
 import 'package:flutter/foundation.dart';
-import 'package:flutter/widgets.dart';
-
-import 'material_state.dart';
-import 'theme.dart';
+import 'package:flutter/material.dart';
```

Comment:
This will cause the material package to be imported recursively. We might need to remove this import:)